tests/network/vmi_networking.go

```diff
@@ -57,6 +57,13 @@ import (
 	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
+const (
+	vmiNetworkingEchoCmd = "echo $?\n"
+	checkingEth0MACAddr  = "checking eth0 MAC address"
+	nechoHello           = "#!/bin/bash\necho 'hello'\n"
```

Comment:
Same as above, this is just a "hello world" placeholder bash script.
Maybe "bashHelloScript"?
I wouldn't be surprised if this was used in other modules as well.